zebra-rpc/src/methods.rs

```diff
@@ -69,6 +70,22 @@ pub trait Rpc {
     #[rpc(name = "getblockchaininfo")]
     fn get_blockchain_info(&self) -> Result<GetBlockChainInfo>;
 
+    /// Returns the total balance of a provided `addresses` in an [`AddressBalance`] instance.
+    ///
+    /// # Parameters
+    ///
+    /// - `addresses`: (array of strings) A list of base-58 encoded addresses.
+    ///
+    /// # Notes
+    ///
+    /// zcashd also accepts a single string parameter instead of an array of strings, but Zebra
+    /// doesn't because lightwalletd always calls this RPC with an array of addresses.
+    ///
+    /// zcashd also returns the total amount of Zatoshis received by the addresses, but Zebra
+    /// doesn't because lightwalletd doesn't use that information.
+    #[rpc(name = "getaddressbalance")]
+    fn get_address_balance(&self, addresses: Vec<String>) -> BoxFuture<Result<AddressBalance>>;
```

Comment:
This doesn't match the expected RPC format, which is one map argument with an "addresses" key. (The above is one list argument). Tested with curl:

```
curl -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","method":"getaddressbalance","params":[{"addresses": ["tmYXBYJj1K7vhejSec5osXK2QsGa5MTisUQ"]}],"id":123}' 127.0.0.1:28232
{"jsonrpc":"2.0","error":{"code":-32602,"message":"Invalid params: invalid type: map, expected a sequence."},"id":123}
```

Reply:
Fixed, thanks!